components/index.ftd

```diff
@@ -0,0 +1,124 @@
+-- import: fastn.com/ftd/utils as appendix
+
+-- ds.page: Blog Template Components
+document-title: fastn Blog Template Template. Build Your Own Website with Ease
+document-description: Simple, easy-to-use blog-template template to build your own documentation website, personal website, or blog. Change colours and typography effortlessly.
+document-image: https://fastn-community.github.io/blog-template/-/fastn-community.github.io/blog-template/assets/blog-template-og-image.jpg
+
+Components are like Lego blocks for building websites. They are pre-designed 
+building blocks that serve specific purposes or display specific content, 
+allowing you to easily create a complete webpage by combining and customizing 
+them.
+
+-- ds.h1: Index
+
+We have listed all blog-template components below, click on a component name to
+learn more about it.
+
+-- appendix.letter-stack:
```

Comment:
@MeenuKumari28 append the `letter-stack` component inside `doc-site` and import it here.